### PR TITLE
Adding colombian foundation

### PR DESCRIPTION
**Pull request by santy2001**
Just adding the Bitcoin Colombia Foundation (non-profit).

### CODE REVIEW

**Comment by santy2001** [comment]
Just following the instructions mentioned on the request  #1351 ,  #1352  .

File: _templates/community.html [hunk 1]
```diff
@@ -104,6 +104,12 @@ <h3 id="canada"><img src="/img/flags/CA.png" alt="Canadian flag">Canada</h3>
   </div>
 </div>
 <div>
+ +  <h3 id="colombia"><img src="/img/flags/CO.png" alt="Bandera de Colombia / Colombian Flag">Colombia</h3>
```
Comment (wbnns) [inline _templates/community.html]: @santy2001 Could you please revise this to `Colombian flag` so that it matches the schema of the other ALT tags?

**Comment by wbnns** [comment]
@santy2001 Hello, just wanted to follow-up on my last message. Could you please [update the ALT tag](https://github.com/bitcoin-dot-org/bitcoin.org/pull/1424#discussion_r92213419)? Once you've done this, all you need to do is rebase. If you have any questions or need any help, please let us know.

**Comment by santy2001** [comment]
Hi,

I already do the change

Thanks, and sorry for be late

File: _templates/community.html (hunk 1 above)
Comment (santy2001) [inline _templates/community.html]: Hi, thats solved

**Comment by wbnns** [comment]
@santy2001 No problem - thank you for making the updates!

Unless others object, this will be merged on Sunday, February 12th.

**Comment by wbnns** [comment]
This is on the site, now.

**Pull request closed without merging**